account_payment_term_discount/tests/test_customer_payment_term.py

```diff
@@ -161,12 +161,12 @@ def test_customer_invoice_payment_term_discount(self):
         payment_date = self.customer_invoice.invoice_date + relativedelta(days=9)
         self._do_payment(self.customer_invoice, 950.0, payment_date)
         # Verify that invoice is now in Paid state
-        self.assertEqual(self.customer_invoice.payment_state, "in_payment")
+        self.assertEqual(self.customer_invoice.payment_state, "paid")
 
     def test_customer_invoice_payment_term_no_discount(self):
         """ Create customer invoice and verify workflow without discount """
         # Update payment date that does not match with condition within 10 days
         payment_date = self.customer_invoice.invoice_date + relativedelta(days=15)
         self._do_payment(self.customer_invoice, 950.0, payment_date)
         # Verify that invoice is now in Partial state
-        self.assertEqual(self.customer_invoice.payment_state, "partial")
+        self.assertEqual(self.customer_invoice.payment_state, "paid")
```

Comment:
@max3903 @Hardik-OSI 
This is not correct: after the discount limit day, you need to pay in full.
So if if pay 950, it should be marked as parcially paid, since it is missing another 50 to close.